share/zm_testagent-bsd

```diff
@@ -16,6 +16,7 @@ load_rc_config $name
 : ${zm_testagent_pidfile="/var/run/zonemaster/${name}.pid"}
 
 export ZONEMASTER_BACKEND_CONFIG_FILE="/usr/local/etc/zonemaster/backend_config.ini"
+#export ZM_BACKEND_TESTAGENT_LOGLEVEL='warning'  # Set this variable to override the default log level
```

Comment:
same remark on `ZM_BACKEND_TESTAGENT_LOGLEVEL`.

Reply:
I fixed this the same way.